crates/store/src/server/mod.rs

```diff
@@ -61,8 +62,9 @@ impl Store {
     ///
     /// Note: this blocks until the server dies.
     pub async fn serve(self) -> Result<(), ApiError> {
+        // Build the gRPC server with the API service and trace layer.
         tonic::transport::Server::builder()
-            .trace_fn(miden_node_utils::tracing::grpc::store_trace_fn)
+            .layer(TraceLayer::new_for_grpc().make_span_with(store_trace_fn))
```

Comment:
Think the default callbacks might suffice for this given we are adding request-related attributes to the span. Note that the defaults ones are debug, not info.

Reply:
I think that's fine yeah